test/test_model.py

```diff
@@ -969,6 +969,28 @@ def test_run_error(self):
         self.assertEqual(str(cm.exception), 'ERROR cannot get status\n')
         self.assertEqual(cm.exception.args[0], 'ERROR cannot get status\n')
 
+    @patch("pwd.getpwuid")
+    @patch("grp.getgrgid")
```

Comment:
It doesn't matter here as we're patching both with the same side effect, but to avoid confusion, these should be  in reverse order to the parameters:
```suggestion
    @patch("grp.getgrgid")
    @patch("pwd.getpwuid")
```

Reply:
Ah, nice catch, thanks!